src/server/identity/db.go

```diff
@@ -32,3 +32,11 @@ CREATE TABLE IF NOT EXISTS identity.config (
 `)
 	return err
 }
+
+// AddTokenExpiryConfig adds expiry fields for token lifespan to the server config
+func AddTokenExpiryConfig(ctx context.Context, tx *sqlx.Tx) error {
+	_, err := tx.ExecContext(ctx, `
+ALTER TABLE identity.config ADD COLUMN
+	id_token_expiry VARCHAR(4096)`)
```

Comment:
maybe this is a trivial question, but is it worth calling out somewhere the format of the Expiry string?

Reply:
@nadegepepin This does add a new configuration option for the identity server we'll need to document - it takes a Go duration string (https://golang.org/pkg/time/#ParseDuration)